ex2.py

```diff
@@ -0,0 +1,6 @@
+def get_positive_numbers():
+    """ gets from the user list of numbers (as str) conver it to int and insert it to a list,
+    run on the list and keeps only positive numbers"""
```

Comment:
Write documentation in the right format - 
python
def count_hi_v4(word: str) -> int:
    """
    The function counts how many times the string "hi" appears in a given string.
    @param word: The given string.
    @return: The amount of times the word "hi" appeared.
    """
    return word.count("hi")